dev-games/aseprite/files/aseprite-1.2.40_ixwebsocket-gcc13.patch

```diff
@@ -0,0 +1,244 @@
+From a5d86afd43920d2401682ecd3ec6879a574845ea Mon Sep 17 00:00:00 2001
```

Comment:
Have you sent this patch upstream? If so, please add a link to the PR/commit. If not, please do ;)

Reply:
Yes, I do: https://github.com/machinezone/IXWebSocket/pull/443

Reply:
Thanks, can you put it at the top of the patch?